app/controllers/idv/otp_delivery_method_controller.rb

```diff
@@ -5,19 +5,17 @@ class OtpDeliveryMethodController < ApplicationController
 
     before_action :confirm_phone_step_complete
     before_action :confirm_step_needed
-    before_action :set_otp_delivery_method_presenter
-    before_action :set_otp_delivery_selection_form
+    before_action :idv_phone # Memoize to use ivar in the view
 
-    def new; end
+    def new
+      analytics.track_event(Analytics::IDV_PHONE_OTP_DELIVERY_SELECTION_VISIT)
+    end
 
     def create
-      result = @otp_delivery_selection_form.submit(otp_delivery_selection_params)
+      result = otp_delivery_selection_form.submit(otp_delivery_selection_params)
+      analytics.track_event(Analytics::IDV_PHONE_OTP_DELIVERY_SELECTION_SUBMITTED, result.to_h)
       if result.success?
-        prompt_to_confirm_phone(
-          phone: @otp_delivery_selection_form.phone,
-          context: 'idv',
-          selected_delivery_method: @otp_delivery_selection_form.otp_delivery_preference
-        )
+        prompt_to_confirm_idv_phone
```

Comment:
Pictured here: Me begging for merge conflicts against #2430

Reply:
Here's a wild idea: what if we got rid of this `otp_delivery_selection_form` altogether?The only thing it's doing is making sure the method is either "sms" or "voice", and the only way it would not be is if someone deliberately manipulated the HTML on the page before submitting the form. How about if we just default to "sms" if something other than "sms" or "voice" is submitted?

Reply:
Or maybe default to not do anything and just display the delivery selection page again.

Reply:
It makes some changes to the idv session in the other PR. The delivery preference gets saved there instead of added as a url param. We could move that logic to the controller, but idk if we want to?

Reply:
Actually, I'm looking at my new code now and I do actually already set it in the controller. Hmm, yeah we can try removing it completely.

Reply:
Though another downside of not have a form is that we don't have a result object to feed to analytics here.

Reply:
@monfresh: Anything to add here?

Reply:
All good. I think we decided to leave this here when we discussed this offline.